src/components/item/item-sliding.scss

```diff
@@ -80,13 +80,7 @@ ion-item-sliding.active-slide {
     display: flex;
   }
 
-  &.active-options-left ion-item-options[side=left] {
-    width: 100%;
-
-    visibility: visible;
-  }
-
-  &.active-options-right ion-item-options:not([side=left]) {
+  &.active-options ion-item-options {
```

Comment:
wow ok I didn't realise. So I'll just merge the selector instead of replacing it